amostra/tests/test_jsonschema.py

```diff
@@ -0,0 +1,35 @@
+import hypothesis_jsonschema
+from hypothesis import HealthCheck, given, settings
+from hypothesis import strategies as st
+
+from amostra.utils import load_schema
+
+sample_dict = load_schema("sample.json")
+# Pop uuid and revision cause they are created automatically
+sample_dict['properties'].pop('uuid')
+sample_dict['properties'].pop('revision')
+# sample_dict['properties']['name']['minLength'] = 1
+sample_dict['required'] = sample_dict['required'][2:]
```

Comment:
I think it would be more readable and robust to explicitly `remove` them, rather than implicitly relying and `uuid` and `revision` being the first two items in the list.